src/pages/resources/fundamentals/object-model/index.md

```diff
@@ -0,0 +1,58 @@
+# Object model
+
+Understanding the object model is the key to finding the object you want to work with.
+
+The following figure is an overview of the InDesign object model. Objects in your publication are arranged in a specific order: frames are on pages, which are inside a document, which is inside the InDesign application object. This hierarchy of objects is referred to as the object model and these objects together make up an InDesign publication.
+
+![InDesign object model](id-om.png)
```

Comment:
@anoopvalomkot  Need an image with a better resolution.

Reply:
@padmkris123 I have emailed Aditya (UX team) for same, you are in cc. If you have any template already in mind, please share the same to Aditya.